app/components/Configuration.js

```diff
@@ -42,6 +42,8 @@ export default class Configuration extends Component<Props> {
   componentDidMount() {
     // $FlowFixMe - flow thinks document.body could be undefined
     document.body.addEventListener('keydown', this.handleKeyDown);
+    // Indicates lifecycle start. Needed so that configuration doesn't autohide anytime it's visible
```

Comment:
tbh, i didn't understand this comment intuitively, so our future selves will have a hard time. you could give it another try or i'll do so once we clear out the issue below